### ISSUE TITLE
[Feature-Request] add progressbar in Cleanup Scans / OCR (Optical Character Recognition)

### ISSUE THREAD

**Issue by Karsto-sys** [Frooodle/Stirling-PDF#149]
Please consider adding a progress bar for the Cleanup Scans / OCR. It would provide users with a visual representation of the progress and improve the overall user experience. A progress bar helps users track the action completion and manage expectations. Additionally, it enhances communication and prioritization when multiple actions are running simultaneously.

Thank you for considering this feature request. It would greatly benefit the application and its users.

**Comment by JefPresotto** [comment]
Seria interessante a barra de progresso em todos os cards, pois indicaria o andamento do processo.

**Comment by Frooodle** [comment]
This feature is planned.

**Comment by ClearSeve** [comment]
I really want this function.

**Comment by jmot205** [comment]
Agreed; this sounds like a great idea. My server runs OCRs quite slowly, so would be good to have an indicator the of progress.

**Comment by linhexu** [comment]
give me this task

**Comment by Frooodle** [comment]
Progress par planned to be picked up
Most likely this will be a "fake" progress bar with a estimated completion time based on filesize/operation

**Comment by b-v-d-e-v** [comment]
We handle large PDF files (500MB, > 1000 pages) too. 
Progress for upload and the processing itself would be really great!

**Comment by dhnnjydahiya** [comment]
@Frooodle Why fake progress bar? We can go ahead with using redis pub-sub technique between two socket.io servers.

Correct me if i am wrong.

**Comment by MuhammadFaisal1521** [comment]
Hi, this is feature planned, 
i can work on this ?
